create-mango/source/app.tsx

```diff
@@ -171,8 +171,6 @@ export default function App({ destination, flags }: Props) {
 	}
 
 	return (
-		<Text>
-			Hello, <Text color="green">"Hello"</Text>
-		</Text>
+		<Wizard/>
```

Comment:
Just return `<Wizard />` in this function.

Reply:
I'm not getting you

Reply:
Updated.